data/pirate jobs.txt

```diff
@@ -2465,6 +2465,58 @@ mission "Cargo Theft (South)"
 		payment 1500000
 		dialog "The pirates of <planet> take the secret cargo from your cargo hold and pay you <payment>."
 
+mission "Capture Smuggler"
+	name "Capture rogue smuggler"
+	minor
+	description `A smuggler absconded with his illegal cargo near <system>. The unknown person he stole it from wants it back. Recover the cargo from the <npc> and return both it and the smuggler to <origin> for <payment>.`
+	to offer
+		random < 15
+		"combat rating" > 2000
+		"reputation: Pirate" > 10
+	source
+		government "Pirate"
+	npc board
+		conversation
+			`After neutralizing the crew and capturing the captain, you have your men sweep the ship. Ultimately they find the stolen cargo in the captain's cabin, hidden behind a false panel. The cargo is a nondescript storage container, but upon closer inspection you see that it's code-locked and covered with interference plating.`
```

Comment:
The player can never be assumed to be in a ship with crew members. For this reason, you shouldn't refer to "your men" sweeping the ship. Perhaps the player could be given a "special gift" from the thick accented man to help dispatch of the crew on your own. 

I'd change this to something along the lines of
`You throw the "special gift" into the cockpit of the ship. A loud bang can be heard, followed by the thud of bodies hitting the floor. After tying up the captain and placing him in your ship, you sweep the <npc> for the cargo. You find the stolen cargo in the captain's cabin, hidden behind a false panel in a small storage container covered in interference plating.`

Then the man could hand you this "special gift" along with the datachip he hands you after accepting.

Also, `npc` blocks typically appear after `on offer` and before `on complete`, especially when the `npc` has some sort of conversation involved. That way when reading the mission in the files, you can read the mission in order instead of needing to jump around.

Reply:
All the other examples in this file and `jobs.txt` at least have `npc` after `to offer` but before `on offer`. Should I change it here anyway? Some other files have it the way you described.

Reply:
Once all the other changes look good I'll move the mission, then.